lib/gollum-lib/macro.rb

```diff
@@ -25,6 +25,10 @@ def html_error(s)
       "<p class=\"gollum-error\">#{s}</p>"
     end
 
+    def active_page()
```

Comment:
just for stylistic unity, would you mind removing the parantheses here? Ruby doesn't need them :) After that I'll merge this PR!